docs/quick-starts/feature-flag-quickstart.md

```diff
@@ -1,11 +1,11 @@
 ---
 slug: /feature-flag-quickstart
-sidebar_position: 3
+sidebar_position: 2
 ---
 
-# Running your first experiment
+# Creating your first feature flag
 
-This 10 minute guide will walk through running your first experiment with Eppo. In the example we'll imagine that we are testing a new checkout page.
+This 10 minute guide will walk through running your first feature flag with Eppo. In the example we'll imagine that we are testing a new checkout page.
 
 While Eppo feature flags can be used for feature gates, kill switches, and targeted rollouts, this guide will focus on using Eppo flags for running randomized experiments.
 
```

Comment:
I guess you can't propose changes on lines that are not part of the diff? 

We should drop this sentence for the feature flagging page:

> Note that if you are using Eppo alongside an existing randomization tool, you can skip right to the [next quick start](/experiment-quickstart).

Reply:
done